File: python-vs-ruby/python/class.py
class Foo:
    def __init__(self):
        self.bar = "bar"

    def baz(self):
        return "Instance method baz: %s" % (self.bar)

    @staticmethod
    def bazbar():
        return "Class method baz"

x = Foo()

print(x.bar)

print(x.baz())

print(Foo.bazbar())

File: python-vs-ruby/ruby/class.rb
class Foo
  attr_accessor :bar

  def initialize
    @bar = "bar"
  end

  def baz
    "Instance method baz: #{@bar}"
  end

  def self.bazbar
    "Class method baz"
  end
end

x = Foo.new

puts x.bar

puts x.baz

puts Foo.bazbar
